### DISCUSSION TITLE
Y3038 Problem and 64-bit Time in EJBCA

### DISCUSSION THREAD

**Discussion by logix11** [Keyfactor/ejbca-ce#672] [Q&A] [answered]
I am configuring EJBCA for the first time, pre-installation. The Y2038 problem is avoided by using 64-bit timing. My OS does support 64-bit, and my java does support 64-bit. How can I check whether my EJBCA supports 64-bit timing or not?
Thank you

**Comment by primetomas** [comment, marked as answer]
[RFC5280](https://datatracker.ietf.org/doc/html/rfc5280#section-4.1.2.5) specifiies date formats for X.509 certificates, handling Y2038. EJBCA is compliant with this.
